app/models/events/move_cancel.rb

```diff
@@ -0,0 +1,24 @@
+class Events::MoveCancel < GenericEvent
+  attr_writer :cancellation_reason
+
+  EVENTABLE_TYPES = %w[Move].freeze
+
+  validates :cancellation_reason, inclusion: { in: Move::CANCELLATION_REASONS }
+  validates :eventable_type, inclusion: { in: EVENTABLE_TYPES }
```

Comment:
since we will be doing this across multiple types, does it make sense to move this to the parent class?

Reply:
Each event will define it's own list of valid eventable types

Reply:
sorry meant the validation itself, if this validation is duplicated across event types, might be better to move to the parent